WMF Framework/CommonStrings.swift

```diff
@@ -276,6 +276,10 @@ public class CommonStrings: NSObject {
     public static let watchlist = WMFLocalizedString("watchlist", value: "Watchlist", comment: "Title for watchlist feature.")
 
     public static let watchlistFilter = WMFLocalizedString("watchlist-filter", value: "Filter", comment: "Title for filter button in watchlist.")
+
+    public static let watchlistByte = WMFLocalizedString("watchlist-byte", value: "byte", comment: "Displayed in watchlist view to indicate a single or no byte change in a revision.")
```

Comment:
@staykids I think you can remove these strings now.

Reply:
Removed - thanks for catching this!